feat: add edge user support

rylancates opened the pull request:
As you can see:
- I don't know the details for Darwin as I don't have a Mac
- I don't know what needs to happen with Edge to get system-wide install working
- I don't know what the policies make steps are for or how to test

maximbaz commented on Makefile:
```diff
@@ -185,6 +185,34 @@ hosts-chrome-user:
+hosts-edge:
+	# THIS DOESN'T WORK
+	@case $(OS) in \
+	Linux)      mkdir -p "/opt/microsoft/msedge/native-messaging-hosts/"; \
```
Instead of the comment, let's just have `Linux)` block commented out too, so that the `make` goal exists, but using it would result in getting an error message, hopefully encouraging community to contribute 😜

maximbaz commented on README.md:
```diff
@@ -167,6 +167,7 @@ If you provided `PREFIX` and/or `DESTDIR` while running `make install`, remember
 | `make hosts-firefox-user`  | Configure browserpass for Firefox browser, for the current user only                 |
 | `make hosts-librewolf-user`| Configure browserpass for Librewolf browser, for the current user only               |
 | `make hosts-chrome-user`   | Configure browserpass for Google Chrome or Opera browsers, for the current user only |
+| `make hosts-edge-user`     | Configure browserpass for Microsoft Edge browser, for the current user only          |
```
In line with the suggestion above, let's add the `hosts-edge` to the README as well 😉

rylancates replied:
Change pushed!

rylancates commented:
I've updated the PR to include the changes you've asked for along with stubs for the policy steps, commented out so that a nice failure is printed for those steps as well. I've updated the readme for both hosts and policies in kind.

maximbaz commented:
Awesome, thanks for your contribution!

rylancates commented:
Thanks for being a kind and helpful maintainer 😎
